### PR TITLE
fix: don't reset height for sbb-textarea

### LINKED ISSUE #862: Bug: Textarea jumps/leads to scrolling on input
**Describe the bug**
when having longer input (many lines) in the textarea it sometimes leads to unintended scrolling.

**To Reproduce**
1. Go to https://angular.app.sbb.ch/public/components/textarea
2. Click on "Showcase"
3. Enter multiple lines
4. Scroll so that only last few lines are visible
5. Enter empty lines and text
6. See that sometimes the scrollbar jumps (not easy to reproduce)

**Expected behavior**
Scrolling should be consistent and not jumping up and down

**Screenshots**
jumping-textarea

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser Chrome  92.0.4515.131 (Offizieller Build) (64-Bit)
 - Version SACL 12.3

**Additional context**
In our project (with perfect-scrollbar) this can always/easily be reproduced!
Maybe related: https://stackoverflow.com/questions/56329625/preventing-textarea-scroll-behaviour-in-chrome-after-newline-added

### PR DESCRIPTION
**Pull request by kyubisation**
Closes #862

### CODE REVIEW

**Comment by jeripeierSBB** [comment]
I think in the merge showcase, the fix does not resolve the problem?
 
https://angular-staging.app.sbb.ch/867-merge/angular/components/textarea/examples

File: src/showcase/styles.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@ body {
 @include sbbLink();
 @include cdk.a11y-visually-hidden();
 @include cdk.overlay();
+@include cdk.text-field();
```
Comment (jeripeierSBB) [inline src/showcase/styles.scss:16]: import seems to be deprecated:

```
// @deprecated Use 'autosize' and 'autofill' instead.
@mixin text-field {
  @include text-field-autosize();
  @include text-field-autofill();
}
```

File: src/angular-public/typography.scss [hunk 2]
```diff
@@ -15,5 +15,6 @@
 @include sbbTable();
 @include cdk.a11y-visually-hidden();
 @include cdk.overlay();
+@include cdk.text-field();
```
Comment (jeripeierSBB) [inline src/angular-public/typography.scss:18]: import seems to be deprecated:

```
// @deprecated Use 'autosize' and 'autofill' instead.
@mixin text-field {
  @include text-field-autosize();
  @include text-field-autofill();
}
```

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]
Maybe we should also enhance the markdown file?

Futhermore I think we should change the PR title to a refactor, because it doesn't really solve the issue, but it's better code

**Comment by kyubisation** [comment]
I updated the markdown files.
I would disagree; Previously the necessary CDK class was missing from the typography, which was causing an issue for the autosizing. This PR fixes that problem.

**Pull request merged**